docker-compose.yml

```
@@ -49,8 +49,8 @@ services:
     depends_on:
       database-migration:
         condition: service_completed_successfully
-      webrequest:
-        condition: service_healthy
+      webdriver:
+        condition: service_healthy  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)